### ISSUE TITLE
intel_gpu_drv: compile error with GCC 8.3.0

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#3332]
When building 'make drivers/gpu' with GCC 8.3.0, the following error occurs:

```
    COMPILE  main.o
In file included from /.../repos/os/src/drivers/gpu/intel/main.cc:37:
/.../repos/os/src/drivers/gpu/intel/mmio.h: In member function ‘void Igd::Mmio::update_context_status_pointer()’:
/.../repos/os/src/drivers/gpu/intel/mmio.h:1224:48: error: unused variable ‘rp’ [-Werror=unused-variable]
    RCS_RING_CONTEXT_STATUS_PTR::access_t const rp = read<RCS_RING_CONTEXT_STATUS_PTR::Read_pointer>();
                                                ^~
```

I'm not sure if the line can be removed or if the 'read<>()' call is still needed. @cnuke?

**Comment by cnuke** [comment]
@cproc to be honest I am not sure if it is needed, it looks superfluous - just remove it.

**Comment by cproc** [comment]
Fixed in c3dd082
